docbrown/db/src/graph_window.rs

```diff
@@ -277,72 +277,88 @@ impl GraphViewInternalOps for WindowedGraph {
             .neighbours_ids_window(v, self.actual_start(t_start), self.actual_end(t_end), d)
     }
 
-    fn vertex_prop_vec(&self, v: VertexRef, name: String) -> Vec<(i64, Prop)> {
+    fn static_vertex_prop(&self, v: VertexRef, name: String) -> Option<Prop> {
+        self.graph.static_vertex_prop(v, name)
+    }
+
+    fn static_vertex_prop_keys(&self, v: VertexRef) -> Vec<String> {
+        self.graph.static_vertex_prop_keys(v)
+    }
+
+    fn temp_vertex_prop_vec(&self, v: VertexRef, name: String) -> Vec<(i64, Prop)> {
         self.graph
-            .vertex_prop_vec_window(v, name, self.t_start, self.t_end)
+            .temp_vertex_prop_vec_window(v, name, self.t_start, self.t_end)
     }
 
-    fn vertex_prop_vec_window(
+    fn temp_vertex_prop_vec_window(
         &self,
         v: VertexRef,
         name: String,
         t_start: i64,
         t_end: i64,
     ) -> Vec<(i64, Prop)> {
-        self.graph.vertex_prop_vec_window(
+        self.graph.temp_vertex_prop_vec_window(
             v,
             name,
             self.actual_start(t_start),
             self.actual_end(t_end),
         )
     }
 
-    fn vertex_props(&self, v: VertexRef) -> HashMap<String, Vec<(i64, Prop)>> {
-        self.graph.vertex_props_window(v, self.t_start, self.t_end)
+    fn temp_vertex_props(&self, v: VertexRef) -> HashMap<String, Vec<(i64, Prop)>> {
+        self.graph.temp_vertex_props_window(v, self.t_start, self.t_end)
     }
 
-    fn vertex_props_window(
+    fn temp_vertex_props_window(
         &self,
         v: VertexRef,
         t_start: i64,
         t_end: i64,
     ) -> HashMap<String, Vec<(i64, Prop)>> {
         self.graph
-            .vertex_props_window(v, self.actual_start(t_start), self.actual_end(t_end))
+            .temp_vertex_props_window(v, self.actual_start(t_start), self.actual_end(t_end))
+    }
+
+    fn static_edge_prop(&self, e: EdgeRef, name: String) -> Option<Prop> {
```

Comment:
Choose between names and keys throughout the project.

Reply:
I'll create a ticket